cmake/thirdparty/SetupThirdParty.cmake

```diff
@@ -115,3 +115,21 @@ if(CMAKE_GENERATOR STREQUAL "Unix Makefiles" OR CMAKE_GENERATOR STREQUAL "Ninja"
     blt_find_executable(NAME        ClangTidy
                         EXECUTABLES clang-tidy)
 endif()
+
+#------------------------------------
+# Code coverage
+#------------------------------------
+if (ENABLE_COVERAGE)
+    # find_executable requires that the executables be enabled
+    set(ENABLE_GCOV ON)
+    set(ENABLE_LCOV ON)
+    set(ENABLE_GENHTML ON)
+    blt_find_executable(NAME        gcov
+                        EXECUTABLES gcov)
+
+    blt_find_executable(NAME        lcov
+                        EXECUTABLES lcov)
+
+    blt_find_executable(NAME        genhtml
```

Comment:
Side comment -- could we please update the documentation for `blt_find_executable` to reflect that it only attempts to find `arg_NAME` when `ENABLE_${arg_NAME}` is `TRUE` ?

https://github.com/LLNL/blt/blob/05ebb179779494f600d9031737888981d7b566ce/cmake/BLTPrivateMacros.cmake#L133-L144

(perhaps in a separate PR if you feel it is off topic for this one).